docs/scenarios-workflow.md

```diff
@@ -0,0 +1,90 @@
+
+# Scenario Tests Guide
+
+## Overview
+
+Our existing scenario tests are under `src\scenarios` in this repo, where each subdirectory contains a test asset that can be combined with a specific set of commands to do measurements. Currently we have scenario tests for [SDK](link), [Crossgen](link), [Blazor](link) and [other scenarios]().
+
+
+## Running scenario tests
```

Comment:
Feel like it might be worthwhile to call out which of the directories in the scenarios folder corresponds to which kind of testing. That way when I want to run weblarge2.0, I know what guide to follow.

Reply:
Yeah the current classification is based on product, but we can definitely add a matrix for asset-specific (aka directory-specific) classification.